KarhooSDK/Service/Loyalty/LoyaltyService.swift

```diff
@@ -20,4 +20,8 @@ public protocol LoyaltyService {
     func getLoyaltyEarn(identifier: String, currency: String, amount: Int, burnPoints: Int) -> Call<LoyaltyPoints>
     
     func getLoyaltyPreAuth(preAuthRequest: LoyaltyPreAuth) -> Call<LoyaltyNonce>
+    
+    func getCurrentLoyaltyStatus(identifier: String) -> LoyaltyStatus?
```

Comment:
We need to keep the names consistent in both Android and iOS

https://github.com/karhoo/karhoo-android-sdk/blob/develop/sdk/src/main/java/com/karhoo/sdk/api/service/loyalty/LoyaltyService.kt

Reply:
The names of pieces of information saved in the UserStore contain "current" on Android as well (See currentUser), but this is not a rule. Due to the fact that the UserStore is accessible from outside of the SDK on Android, but not on iOS, "current" is needed on iOS to differentiate between information in the UserStore and information retrieved from the server.